src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,55 @@
+@import "../utils/variables";
+@import "../utils/mixins";
+
+.clock {
```

Comment:
Don't put every value into a variable, this reduces the code readability. Generally you should put into css variables only constant values that repeat across multiple files so they can be easily accessed and changed without breaking the consistency of the site / layout. For example things like font sizes or colors most likely need to be the same for a feature or whole app so it's a good idea to put them into variables, but things like specific size of one element don't qualify for a variable.